Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -12,12 +12,17 @@ import BraintreeCore
     /// Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
     public var offerCredit: Bool
 
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    public var userAuthenticationEmail: String?
```

Comment:
I don't think this will work in the web based PayPal vault flows currently - it should only live in the NXO module